conf/default.php

```diff
@@ -18,7 +18,6 @@
 $conf['doLogoChangesByNamespace']       = 0;
 
 $conf['_plugins']                       = '';
-$conf['defaultAddNewPage']              = 'NEWPAGE>.';
+$conf['defaultAddNewPage']              = '';
```

Comment:
Actually no, according to the plugin's documentation, the default pattern is {{NEWPAGE}}. So I have arranged the code so that the default would be shown as {{NEWPAGE}}. Also tested to make sure this was accurate. The '>' and pattern is only required if you want to specify a namespace. So you either put a namespace in that field or you don't. If the user puts a namespace then '>' will be prepend to the namespace.